app/TonerLandService.py

```diff
@@ -0,0 +1,95 @@
+import requests
+import ExcelObject
+import json
+import re
+from bs4 import BeautifulSoup
+
+
+class TonerLandService:
+    def __init__(self):
+        print('Initializing')
+
+# baseURl + Series hardcoded array webscrape or something
+# for each series scrape source
+# 
+# 
+#
+ 
+
+    def createUrlArray(self):
+        baseUrl = 'http://www.tonerland.com/brother/'
+        seriesArray = ['dcp-series.html', 'fax-series.html', 'hl-series.html', 'intellifax-series.html', 'mfc-series.html'] 
+        PartialURLArray=[]
+        for s in seriesArray:
+            PartialURLArray.append(baseUrl+s) 
+        return PartialURLArray
+            
+
+    def GetResults(self):
+        response = requests.get('file:///base.html',timeout=5)
+        return response.text
+
+    def parseItemContainer(self, container):
+        santizing=container.split()
+        return santizing
+
+    def find1stKeyword(self,word):
+        firstWord='var'
+        
+        if word==firstWord:
+            return True
+        else:
+            return False
+    
+
+    def find2ndKeyword(self,word):
+        secondWord='models'
+
+        if word==secondWord:
+            return True
+        else:
+            return False
+
+    def findURLKeyword(self,word):
+        urlKeyword='url:'
+        if word==urlKeyword:
+            return True
+        else: 
+            return False
+
+    def containsKeyWord(self,array):
+        

```

Comment:
What's the purpose of this function? From the looks of it, it looks like it's supposed to do a check and return an array of URLs, so the function name throws me off a bit. Usually if it starts with "contains" or "is" or "has" it says to me that it's returning a boolean.

Reply:
At first i was thinking it would just return a bool but then as I was getting no progress, i just made it into a function that would do more things.

Reply:
Made it back into contains now that it works